rpc/core/tx.go

```diff
@@ -101,5 +103,29 @@ func TxSearch(ctx *rpctypes.Context, query string, prove bool, page, perPage int
 		}
 	}
 
+	if len(apiResults) > 1 {
+		for _, item := range apiResults {
+			if item != nil {
+				apiResults = append(apiResults, item)
+			}
+		}
+
+		if orderBy == "desc" {
```

Comment:
```go
switch orderBy {
case "desc":
...
case "asc":
...
default:
panic(fmt.Sprintf("Unknown orderBy param: %s", orderBy))
}
```

Reply:
@melekes actually this will create issue because by default we wanted to keep "asc" order but, if we will use switch case then if someone will not specify the correct order then the transaction will be unordered.